NewBank/newbank/server/Account.java

```diff
@@ -13,5 +13,16 @@ public Account(String accountName, double openingBalance) {
 	public String toString() {
 		return (accountName + ": " + openingBalance);
 	}
-
+    public double getBalance() {
+		return (openingBalance);
+	}
+	public String getName() {
+		return this.accountName;
+	}
+	public void addMoney(double amount) {
+		this.openingBalance += amount;
```

Comment:
I think that the name of this variable could be changed to currentBalance? What do you think?
But not necessarily in this ticket, I know it was there already :)

Reply:
I agree - it was really annoying for me to remember it was called this! I'll add it to the refactor ticket I raised :)